src/gui/models/modifyChargesModel.h

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2024 Team Dissolve and contributors
+
+#pragma once
+
+#include "classes/species.h"
+#include <QObject>
+
+class ModifyChargesModel : public QObject
+{
+    Q_OBJECT
+
+    Q_PROPERTY(double scaleValue READ scaleValue NOTIFY valueSet)
+    Q_PROPERTY(double smoothValue READ smoothValue NOTIFY valueSet)
+    Q_PROPERTY(int sigFigValue READ sigFigValue NOTIFY valueSet)
+
+    public:
+    ModifyChargesModel();
+    ~ModifyChargesModel();
+
+    // Enum type to differentiate between operations for mofiying charges
+    enum Modify { Scaling, Smoothing, ReduceSigFig };
```

Comment:
Just a naming suggestion to aid code readability:
```suggestion
    enum ModifyBy { Scaling, Smoothing, ReduceSigFig };
```